### PR TITLE
feat: redirect v1 URL paths to v2

### PR DESCRIPTION
**Pull request by bonustrack**
Add redirections from v1 URL paths like:

- `/#/safe.eth` to v2 path `/#/s:safe.eth`
- `/#/delegate/safe.eth` to v2 path `/#/s:safe.eth/delegates`
- `/#/safe.eth/proposals` to v2 path `/#/s:safe.eth/proposals`
- `/#/safe.eth/about` to v2 path `/#/s:safe.eth`

Also update snapshot.org URLs to point to v1.snapshot.box and testnet.v1.snapshot.org

### CODE REVIEW

File: apps/ui/src/routes/index.ts [hunk 1]
```diff
@@ -32,4 +33,34 @@ const router = createRouter({
...
+    if (rest && !/^\/about$/.test(rest)) {
+      redirectPath += rest;
+    }
+  }
+
+  // Match and redirect paths like "/delegate/safe.eth" to "/s:safe.eth/delegates"
+  const delegateMatch = to.path.match(/^\/delegate\/([^:\/]+?\.[^:\/]+)$/);
+  if (delegateMatch) {
+    const domain = delegateMatch[1];
+    redirectPath = `/${metadataNetwork}:${domain}/delegates`;
```
Comment (wa0x6e) [inline apps/ui/src/routes/index.ts:55]: This will not work for testnet

Comment (bonustrack) [reply apps/ui/src/routes/index.ts:55]: Why not?

Comment (bonustrack) [reply apps/ui/src/routes/index.ts:55]: Hum, we don't support delegates API on testnet?

Comment (wa0x6e) [reply apps/ui/src/routes/index.ts:55]: Ah no, apologies, was thinking of something else

**Review by wa0x6e — APPROVED** [review]
tAck

**Comment by wa0x6e** [comment]
Links like strategy and playground will just redirect to v2, and stay there.

Shouldn't they also be handled, so they redirect to v1.snapshot.box ?

File: apps/ui/src/routes/index.ts [hunk 2]
```diff
@@ -32,4 +33,41 @@ const router = createRouter({
...
+  }
+
+  // Match and redirect "/strategy/*" or "/playground/*" to v1.snapshot.box
+  if (to.path.startsWith('/strategy/') || to.path.startsWith('/playground/')) {
```
Comment (wa0x6e) [inline apps/ui/src/routes/index.ts:59]: Maybe also `/ranking` ?

Comment (bonustrack) [reply apps/ui/src/routes/index.ts:59]: It's fine, no one knows about that page

**Pull request merged**